### PR TITLE
Styles bundle rows

### PR DESCRIPTION
**Pull request by yipenghe**
#1638 & #1613 
Contains all changes from fix-functions, making the base branch fix-functions to see difference easier.

Check out the bundle row style, it's cool!!!
Screenshot from 2019-11-12 00-47-45

### CODE REVIEW

File: frontend/src/components/worksheets/items/TableItem/BundleRow.js [hunk 1]
```diff
@@ -122,7 +123,7 @@ class BundleRow extends Component {
             if (col === 0) {
                 url = baseUrl;
                 checkBox = <Checkbox
-                                icon={<CheckBoxOutlineBlankIcon fontSize="small" />}
+                                icon={<CheckBoxOutlineBlankIcon color={this.props.focused || this.state.hovered? 'action': 'disabled'} fontSize="small" />}
```
Comment (percyliang) [inline frontend/src/components/worksheets/items/TableItem/BundleRow.js]: nit: space before `?`

File: frontend/src/components/worksheets/items/TableItem/TableItem.js [hunk 2]
```diff
@@ -104,7 +105,7 @@ class TableItem extends React.Component<{
                                 checked={this.state.checked}
                                 onChange={this.handleSelectAllClick}
                                 value="checked"
-                                icon={<CheckBoxOutlineBlankIcon fontSize="small" />}
+                                icon={<CheckBoxOutlineBlankIcon color={this.state.hovered? 'action':'disabled'} fontSize="small" />}
```
Comment (percyliang) [inline frontend/src/components/worksheets/items/TableItem/TableItem.js]: ditto, and space between `:`

**Review by percyliang — APPROVED** [review]
This looks great!  One remaining thing is that the row that the cursor is on has the blue bar, which causes a larger indent than a non-selected row...which causes the rows to shift around a bit.  Can we make the indent the same to avoid the jumpiness?

**Comment by yipenghe** [comment]
> This looks great! One remaining thing is that the row that the cursor is on has the blue bar, which causes a larger indent than a non-selected row...which causes the rows to shift around a bit. Can we make the indent the same to avoid the jumpiness?

Added a transparent border to non-focused rows, it won't jump around now

**Comment by yipenghe** [comment]
Force pushed to make changes here clean and only contain the styling part

**Pull request merged**